content/blog/2022/10/camel-karavan-roadmap/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Karavan Roadmap 2023"
+date: 2022-10-10
+authors: [mgubaidullin]
+categories: ["Tooling", "Karavan"]
+preview: "Karavan Roadmap 2023"
+---
+
+Karavan is an Integration Toolkit for Apache Camel aimed to increase developer performance through the visualization of routes, integration with runtimes and pipelines for package, image build and deploy to kubernetes out-of-the-box.
+
+![karavan-clouds](karavan-clouds.png)
+
+Apache Camel Karavan is celebrating its first anniversary. October 4, 2021 a new repository was created and the first prototype of Karavan Designer was pushed there.
+
+Today we have the Karavan VS Code extension and Cloud-native application for Kubernetes/OpenShift. Karavan supports Enterprise Integration Patterns, REST DSL (including import of OpenAPI and generation of routes), Camel Components and Kamelets. Cloud application comes with Tekton task and pipeline for package, build and deploy.    
+
+After only one year Karavan has become a toolkit used by thousands developers and integrators. 
+However we still have a lot to do to make Apache Camel Karavan the best open-source integration solution.
+
+# Roadmap
+By the next anniversary we would like to have the following features implemented.
+The plan sounds quite optimistic and ambitious but it will help to keep the pace.
+And we welcome and encourage contributors to join the journey.
+
+## Visual debugging and tracing
+Karavan users can already  package, build and deploy integrations in one click.
+However a real integration development process is iterative and requires debugging.
+
+Visual debugging process will be implemented, so users will be able stopping/stepping running routes and have access to exchange data (body, headers, properties)  
+
+## Monitoring and dashboards
+We are planning to create a Dashboard page in the Karavan cloud application.
+
+Dashboard should show all Deployments, Pods, Pipelines and camel-context health-check information associated with Camel projects. 
+Camel projects are projects labeled with `app.openshift.io/runtime=camel` or `app.openshift.io/runtime=camel` and not limited to Karavan-created projects.
```

Comment:
Can we remove this as its an implementation detail, and we should use a more generic label that is not openshift specific.